tf-dataflow/real_data_loader.py

```diff
@@ -0,0 +1,265 @@
+#!/usr/bin/python3
+# https://github.com/tensorflow/tpu/blob/master/tools/datasets/jpeg_to_tf_record.py
+# https://www.tensorflow.org/tutorials/load_data/tfrecord
+
+from __future__ import print_function
+
+import _thread
+import glob
+import json
+import logging
+import os
+import random
+import sys
+
+import tensorflow.compat.v1 as tf
+
+tf.disable_eager_execution()
+
+LOG = logging.getLogger()
+logging.basicConfig(level=logging.DEBUG)
+LOG.addHandler(logging.StreamHandler())
+
+
+_train_val_ratio = 0.8
+_record_size_max_mb = 10.0
+_bytes_in_mb = 1000000.0
+_data_base_path = os.path.join("/disk", "qi")
+_run_id = "downscale_real_data"
+_data_path = os.path.join(_data_base_path, _run_id)
+_out_path = os.path.join(_data_base_path, "downscale_real_tfrecord")
+_concurrency = 10
+if not os.path.exists(_data_path):
+    raise ValueError("Data path does not exist: {}".format(_data_path))
+
+
+def main():
```

Comment:
There are multiple steps here in the main function. Can you break it into smaller functions to improve code readability and maintainability?

Reply:
The steps are logically connected but added comments for readability.